uwmac0068.xml

```diff
@@ -523,34 +1170,236 @@
         </c02>
         <c02>
           <did>
+            <unittitle>Speaker and Colloquium Series, 2006</unittitle>
+            <container type="box" label="Box">8</container>
+            <container type="folder" label="Folder">2</container>
+          </did>
+        </c02>
+        <c02>
+          <did>
+            <unittitle>Speaker Series, Gary Hoover, 2006</unittitle>
+            <container type="box" label="Box">8</container>
+            <container type="folder" label="Folder">3</container>
+          </did>
+        </c02>
+        <c02>
+          <did>
             <unittitle>Spring Newsletter, 1983</unittitle>
             <container type="box" label="Box">4</container>
             <container type="folder" label="Folder">13</container>
           </did>
         </c02>
         <c02>
           <did>
+            <unittitle>Statistical Reports, 1996-1999</unittitle>
+            <container type="box" label="Box">5</container>
+            <container type="folder" label="Folder">17</container>
+          </did>
+          <accessrestrict>
+            <p>Restricted</p>
+          </accessrestrict>
+        </c02>
+        <c02>
+          <did>
+            <unittitle>Steering Committee, 1977-2011</unittitle>
+            <container type="box" label="Box">8</container>
+            <container type="folder" label="Folder">4-10</container>
+          </did>
+        </c02>
+        <c02>
+          <did>
+            <unittitle>Student Affairs Committee, 1988-2010</unittitle>
+            <container type="box" label="Box">5</container>
+            <container type="folder" label="Folder">18-20</container>
+          </did>
+          <accessrestrict>
+            <p>Restricted</p>
+          </accessrestrict>
+        </c02>
+        <c02>
+          <did>
+            <unittitle>Student Committee, 2006-2007</unittitle>
+            <container type="box" label="Box">5</container>
+            <container type="folder" label="Folder">21</container>
+          </did>
+          <accessrestrict>
+            <p>Restricted</p>
+          </accessrestrict>
+        </c02>
+        <c02>
+          <did>
+            <unittitle>Student Forum</unittitle>
```

Comment:
I'd frankly prefer to see these in chronological order rather than alphabetic-- This way doesn't really facilitate access